src/exchange_adapters/whitebit.ts

```diff
@@ -5,9 +5,9 @@ import { Exchange } from '../utils'
 export class WhitebitAdapter extends BaseExchangeAdapter {
   baseApiUrl = 'https://whitebit.com/api/v4/public/'
   readonly _exchangeName = Exchange.WHITEBIT
-  // Cloudflare Inc ECC CA-3
+  // whitebit.com - validity not after: 19/09/2023, 01:59:59 CEST
```

Comment:
hmm.. this is expiring in 12 days already, I'd propose for this one we keep the old one and create an issue to check whitebit again in 2 weeks? To not update it not and then have it expire right after.

Reply:
yeah sounds good